zerver/lib/cache.py

```diff
@@ -73,7 +73,7 @@ def remote_cache_stats_finish() -> None:
     remote_cache_total_time += time.time() - remote_cache_time_start
 
 
-def get_or_create_key_prefix() -> str:
+def get_or_create_key_prefix() -> str:  # nocoverage
```

Comment:
I agree with `nocoverage` here, since it's structurally test infrastructure conditionals.